traffic_ops/app/lib/UI/DeliveryService.pm

```diff
@@ -152,12 +153,7 @@ sub get_example_urls {
 				}
 			}
 			elsif ( $re->{type} eq 'PATH_REGEXP' ) {
-				if ( defined( $example_urls[ $re->{set_number} ] ) ) {
-					$example_urls[ $re->{set_number} ] .= $re->{pattern};
-				}
-				else {
-					$example_urls[ $re->{set_number} ] = $re->{pattern};
-				}
+				push(@example_urls, $re->{pattern});
```

Comment:
It messed up when both "HOST_REGEXP" and "PATH_REGEXP" configured. Root cause:

1. Duplicated "set_number" is not allowed in DS configuration in GUI. So not "PATH_REGEXP" will share the same "set_number" with "HOST_REGEXP".
2. "$re->{set_number}" is not the index of "@example_urls". See the "push( @example_urls, $https_url );" code above. So "$example_urls[ $re->{set_number} ] .= $re->{pattern};" may mess up with any other "HOST_REGEXP".